packages/shared/openapi/externalapi/openapi.yaml

```
@@ -80,6 +80,43 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /.well-known/host-meta:
+    get:
+      operationId: getHostMeta
+      tags: 
+        - HostMeta
+      summary: get host meta
+      description: get host meta
+      responses:
+        200:
+          description: success
+          content:
+            application/xml:
+              schema:
+                type: string
+                example: |
+                  <?xml version="1.0"?>
+                  <XRD xmlns="http://docs.oasis-open.org/ns/xri/xrd-1.0">
+                      <Link rel="lrdd" type="application/xrd+xml" template="https://siren.capslock.dev/.well-known/webfinger?resource={uri}" />
```

参考にしたサイトのままになっていそうなので、変更お願いします:pray:

```suggestion
                      <Link rel="lrdd" type="application/xrd+xml" template="https://stlatica.hoge/.well-known/webfinger?resource={uri}" />
```